src/llmcompressor/pipelines/layer_sequential/helpers.py

```diff
@@ -126,3 +131,27 @@ class EarlyStopException(Exception):
 
     _args: Tuple[Any, ...]
     _kwargs: Dict[str, Any]
+
+
+def maybe_inject_pos_embeddings(
+    output: Dict[str, Any],
+    next_layer: Module,
+    inputs: Dict[str, Any],
+) -> Dict[str, Any]:
+    """
+    As of https://github.com/huggingface/transformers/pull/34858, positional embeddings
```

Comment:
annoying

Reply:
This comment? The transformer changes? Both?

Reply:
Transformer change lol

Reply:
Yes, I wish they had kept the backwards compatibility though